source/develop/release-management/features/virt/headless-vm.html.md

```diff
@@ -0,0 +1,92 @@
+
+
+In case of a Headless VM, the "Graphics Column" and the "General" sub-tab displays the "Graphics protocol" and "Video Type" values as "None" and invoking a graphic console and using the "Console Options" dialog are disabled.
+
+Under the "VM devices" sub-tab no video or graphic devices are displayed in case of a Headless VM.
+
+Those changes are relevant also for the VM Template, VM Pool and Instance Type dialogs and the Headless Mode configuration is supported for all actions relevant for those entities including create a VM from a Template and create a Template from a VM.
+
+An option of "Headless mode" was added to the Console section of the "Run Once" dialog and the user can choose how to run the first run of a VM (headless or not): as non-Headless with VNC graphic protocol, as Non-Headless with SPICE graphic protocol or as Headless.
+
+#### Backend
+
+No new property was added to the VM and marking a VM as Headless is done by using existing properties. 
+In case of Headless VM / Template / Pool / Instance Type:
+1. The VM (VMStatic object) is set the defaultDisplayType property to "none" value.
+2. The VM (VMDynamic object) is set the graphicsInfos property (spice_port, spice_ip, vnc_port, vnc_ip, spice_tls_port etc) to null values.
+3. All Video and graphic devices are removed for that VM.
+
+#### DB
+
+No changes in the DB
+
+### Status
+
+*   Target Release: Ovirt 4.1
+*   Status: All is done except REST API support for this new feature (Backward compatibility is fully supported for REST API in Ovirt 4.1).
+*   REST API support Target Release: Ovirt 4.2
+
+### Limitations
+
+*   The ability to access the Headless VM and obtaining the VM IP are not automatically supported -
+In case a VM is currently running in Ovirt in a Headless mode, without a serial console enabled and without Guest-Agent installed, the user won't be able to obtain an access or login to that VM and also won't be able to check for the VM IP via the Ovirt Engine. 
+There is no warning for such cases in which no guest-agent is running or no serial console enabled and no ssh key installed and it's the user responsibility to manage it in advance (either by enabling the serial console or by manage the VM on it's first run via the "run Once" dialog which enables to run a Headless VM as non-Headless only for the first run).
+
+*   Switching from Headless mode to non-Headless mode for a running VM requires a VM restart (since devices should be loaded or omitted) and this may be a limitation for stateless VMs.
```

Comment:
this can be dropped - in general, next run configuration are irrelevant for stateless VMs. I don't see anything unique in the headless property